CHANGELOG.md

```diff
@@ -5,17 +5,14 @@
 - Updated various types to specify DSNP ids instead of string
 - Updated various types to specify HexString instead of string
 - Updated all instances of keccak256 to use hash utilities instead
-- Added hash() and getHashGenerator() utilitiy functions
 - Moved "requireGet" config methods back into the SDK core
 - Removed put method from the store interface in favor of putStream everywhere
 - Updated activityContent types to match spec
 - Updated activityContent generators to match new types
-- Added type check and validation functions to activityContent module
-- Added type check and validation functions to announcements module
 - Updated content porcelain to accept proper activityContent objects and validate them
 - Renamed activityPub -> activityContent
-- Renamed DSNPId to DSNPAnnouncementId to avoid confusion with DSNPUserId
-- Renamed uri -> url everywhere
+- Renamed DSNPId to DSNPAnnouncementUri to avoid confusion with DSNPUserId
```

Comment:
Given that we all caps `DSNP`, shouldn't we caps `Uri` as well? So this should be `DSNPAnnouncementURI` everywhere?

Reply:
fixed